src/Behat/Testwork/Output/Printer/Factory/FileOutputFactory.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Testwork\Output\Printer\Factory;
+
+use Behat\Testwork\Output\Exception\MissingOutputPathException;
+use Symfony\Component\Console\Output\StreamOutput;
+
+final class FileOutputFactory extends OutputFactory
```

Comment:
This is an output factory which creates an output stream directed to a single file